### PR TITLE
Standardized and QOL Imporvements

### PR DESCRIPTION
**Pull request by Matt07211**
Add CONTRIBUTERS.md
Typo fix
Code style changes to bring it more inline with current plugins

These changes do not impact how the plugin functions

### CODE REVIEW

File: Jellyfin.Plugin.Anime/Jellyfin.Plugin.Anime.csproj [hunk 1]
```diff
@@ -2,20 +2,22 @@
...
     <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>Jellyfin.Plugin.Anime</RootNamespace>
     <AssemblyVersion>1.0.0</AssemblyVersion>
     <FileVersion>1.0.0</FileVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <None Remove="Configuration\configPage.html" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.2.0.1" />
+    <PackageReference Include="Jellyfin.Model" Version="10.2.0.1" />
```
Comment (Bond-009) [inline Jellyfin.Plugin.Anime/Jellyfin.Plugin.Anime.csproj]: This isn't explicitly needed, as `Jellyfin.Controller` already includes it.

**Review by Bond-009 — APPROVED** [review]

File: Jellyfin.Plugin.Anime/Jellyfin.Plugin.Anime.csproj (hunk 1 above)
Comment (Matt07211) [reply Jellyfin.Plugin.Anime/Jellyfin.Plugin.Anime.csproj]: But I don't see it hurting anything leaving it in there either, the plugin template has it, so if you feel like that shouldn't be standard then we should probably change that

Comment (cvium) [reply Jellyfin.Plugin.Anime/Jellyfin.Plugin.Anime.csproj]: We are all about reducing unnecessary cruft though ;)

Comment (Matt07211) [reply Jellyfin.Plugin.Anime/Jellyfin.Plugin.Anime.csproj]: I'll see about dropping the line if I get the chance

**Pull request merged**